lib/common/common/src/flags.rs

```diff
@@ -35,6 +41,7 @@ impl Default for FeatureFlags {
         FeatureFlags {
             all: false,
             payload_index_skip_rocksdb: true,
+            payload_index_skip_mutable_rocksdb: false,
```

Comment:
`payload_index_skip_mutable_rocksdb: false,`

Have you tried setting this to `true` and run all tests on our CI?

Reply:
> IIRC it is true in tests already, 

Do you mean that all unit test run with all the features enabled?
What about integration tests.

I believe you told me once that the CI benchmark uses all feature flag, right?